command/bridge/deploy/params.go

```diff
@@ -13,6 +14,7 @@ const (
 	deployerKeyFlag = "deployer-key"
 	jsonRPCFlag     = "json-rpc"
 	erc20AddrFlag   = "erc20-token"
+	txTimeoutFlag   = "tx-timeout"
```

Comment:
Why not putting to some even more common place (such as `"github.com/0xPolygon/polygon-edge/command/helper"`), export it and reuse it from there? Basically it should be next to the `TxTimeoutDesc`.